src/osversion/VersionTranslator.cpp

```diff
@@ -8310,6 +8306,24 @@ namespace osversion {
         m_refactored.push_back(RefactoredObjectData(object, newObject));
         ss << newObject;
 
+      } else if ((iddname == "OS:Coil:Heating:Gas") || (iddname == "OS:Coil:Heating:Gas:MultiStage")
+                 || (iddname == "OS:Coil:Heating:Gas:MultiStage:StageData") || (iddname == "OS:Coil:Heating:Desuperheater")) {
```

Comment:
Doesn't seem to conflict with the existing fuelTypeRenames and crankcaseCoilWithIndex, so that's good.